src/test/java/org/example/AppTest.java

```diff
@@ -51,4 +50,31 @@ public void test_should_sort_passed_list(){
         //THEN
         Assertions.assertLinesMatch(expected, actual);
     }
+
+    /**
+     * Test throwing IllegalArgumentException in method sortedList
```

Comment:
Please add more meaningful comment. E.g. Test should throw @ IllegalArgumentException if no parameters have been passed to the method.